src/main/scala/ai/privado/entrypoint/ScanProcessor.scala

```diff
@@ -173,7 +176,37 @@ object ScanProcessor extends CommandProcessor {
                             language = Language.withNameWithDefault(pathTree.last)
                           )
                         )
-                        .filter(filterSystemConfigByLang)
+                        .filter(filterSystemConfigByLang),
+                      auditCollection = configAndRules.auditCollection
```

Comment:
What are these 3 new rules and will this be part of Privado repo?

Reply:
There three new rule is for audit dependency only. so to identify which dependency belong to which category (Collection, Web client, Utility) we need rules. Yes, we also need to define the rule inside Privado repo.